src/collections/_documentation/clients/go/usage.md

```diff
@@ -46,11 +46,11 @@ To form a `Packet`, you can use `Packet` type directly, or `NewPacket` and `NewP
 
 ```go
 packet := &raven.Packet{
-  Message: "Hand-crafted event"
-  Extra: &raven.Extra{
-    "runtime.Version": runtime.Version(),
-    "runtime.NumCPU": runtime.NumCPU()
-  }
+    Message: "Hand-crafted event",
+    Extra: &raven.Extra{
+      "runtime.Version": runtime.Version(),
+      "runtime.NumCPU": runtime.NumCPU(),
```

Comment:
This seems wrong. The outer level has now indentation=4 while this line is indented with 6 spaces.

Reply:
Thanks! I fixed the indention.